src/phoebus_driver.cpp

```diff
@@ -1019,6 +1020,39 @@ TaskListStatus PhoebusDriver::MonteCarloStep() {
 void UserWorkBeforeOutput(MeshBlock *pmb, ParameterInput *pin) {
   auto tracer_pkg = pmb->packages.Get("tracers");
   bool do_tracers = tracer_pkg->Param<bool>("active");
+  bool is_stellarcollapse = (pin->GetString("eos", "type") == "StellarCollapse");
+
+  if (is_stellarcollapse) {
```

Comment:
Would it be preferable to just output entropy for all EOS? Might not be so useful for ideal gas, but if we end up running with e.g.m Helmholtz, we might want it then.

Reply:
Good point! I forgot that entropy was always available now. Yes, probably can just call it from arbitrary EOS directly